src/main/java/org/example/models/User.java

```diff
@@ -0,0 +1,13 @@
+package org.example.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor @NoArgsConstructor @Getter @Setter
+public class User {
+    String username;
+    String password;
```

Comment:
what is the password here for? Do we need it anywhere?

Reply:
Do not need, removed